src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Version>1.0.21</Version>
```

Comment:
Isn't it better to  bump version in its own PR (for next time)

Reply:
Maybe we can do it in separate PRs, but it doesn't change much having it in any of the PRs that comes in between releases. It's just so the draft-release is bumped.